DicomRtImportExport/Logic/vtkSlicerDicomRtReader.cxx

```diff
@@ -182,7 +322,7 @@ class vtkSlicerDicomRtReader::vtkInternal
   public:
     ChannelEntry()
     {
-      Number = -1;
+      Number = 0;

```

Comment:
Please explain why you changed this. -1 is an obvious "uninitialized" state, while 0 is valid, so this changes the way a newly instantiated object can be handled.